tests/etl/test_transformers.py

```
@@ -0,0 +1,78 @@
+import pathlib
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)